Changelog.md

```diff
@@ -19,6 +19,7 @@ Bugfixes:
  * Type system: Disallow ``var`` declaration with empty tuple type.
  * Type system: Correctly convert function argument types to pointers for member functions.
  * Type system: Move privateness of constructor into AST itself.
+ * Type system: Support scientific notation in numbers (e.g. ``2e8`` and ``200e-2``).
```

Comment:
The actual code is inside the type checker, but perhaps to the user, this looks more like a parser thing? Oh and is it a bugfix or a feature?